src/Parser/AfflictionWarlock/Modules/Talents/EmpoweredLifeTap.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatNumber, formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+import getDamageBonus from '../WarlockCore/getDamageBonus';
+
+const ELT_DAMAGE_BONUS = 0.1;
+
+class EmpoweredLifeTap extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  bonusDmg = 0;
+  ELTuptime = 0;
```

Comment:
Properties should start with lowercase characters.

Reply:
Will fix in next PR :)